embulk-core/src/main/java/org/embulk/exec/GuessExecutor.java

```diff
@@ -135,7 +147,9 @@ public ConfigDiff guessParserConfig(Buffer sample, ConfigSource inputConfig, Con
         final GuessExecutorTask task = loadGuessExecutorTask(execConfig);
         guessPlugins.addAll(task.getGuessPlugins());
         guessPlugins.removeAll(task.getExcludeGuessPlugins());
-        final int guessParserSampleBufferBytes = task.getSampleBufferBytes();
+        final int guessParserSampleBufferBytes =
+                task.getSampleBufferBytes().orElse(this.systemGuessSampleBufferBytes.orElse(DEAULT_SAMPLE_BUFFER_BYTES));
+        System.out.println(guessParserSampleBufferBytes);
```

Comment:
nit: not descriptive

Reply:
Ah, really. It was just remaining when I was in tries-and-errors. Thanks for the good catch! Removed it in 67d7bc22c998faae5b9c05865591883ad05f1470.